### PR TITLE
chore(client): ts migrate builders.js

### PR DESCRIPTION
**Pull request by DerrykBoyd**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

ref #43778

### CODE REVIEW

File: client/src/templates/Challenges/utils/build.ts [hunk 1]
```diff
@@ -19,15 +19,17 @@ import {
 } from './frame';
 import createWorker from './worker-executor';
 
+export interface ConcatHTMLOptions {
+  required: { src: string; link?: string }[];
+  template?: string;
+  contents?: string;
+}
+
 const _concatHtml = ({
```
Comment (ShaunSHamilton) [inline client/src/templates/Challenges/utils/build.ts:28]: This can be deleted, and its references can be replaced with the import.

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute.

If I am not mistaken, the reason the `_concatHtml` function was created was to facilitate the TS migration of the `build.ts` file without needing to migrate `builders.js`.

However, now that the `builders.js` file is migrated, we can remove the intermediate function.

**Review by ShaunSHamilton — APPROVED** [review]
Thank you for making that change.

This LGTM 👍

**Comment by DerrykBoyd** [comment]
> Thank you for making that change.
> 
> This LGTM 👍

No Problem!

Should we remove the `waiting update` label for this now?

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Pull request merged**